pom.xml

```diff
@@ -1621,6 +1628,12 @@
           <artifactId>smallrye-open-api-maven-plugin</artifactId>
           <version>${version.io.smallrye.openapi.core}</version>
         </plugin>
+
+        <plugin>
+          <groupId>org.wildfly.plugins</groupId>
+          <artifactId>wildfly-jar-maven-plugin</artifactId>
+          <version>${version.bootable}</version>
+        </plugin>
```

Comment:
I guess we need to remove below thorntail plugin too

Reply:
Sure. I don't think anyone should be using it.

Reply:
don't forget on this to remove it  ;)